joomla/joomla-cms
# [4.3][GUIDEDTOURS] Move code from helper to model

Pull Request for Issue # .

### Summary of Changes
This PR move the code which is used to update language of steps belong to a tour from Helper to Model for two reasons:

- In model, we have database object and ready to use, unless in Helper, we have to use Factory::getDbo() to get database object (we are trying to move a way from using the static methods from Factory)
- I believe that a **doing something operation**, especially update data from database should be a task of model, not something for Helper

### Testing Instructions
1. Use Joomla 4.3 nightly build or a Joomla 4.3 installation which has Guided Tours available
2. Edit a tour, save it, make sure nothing broken

### Actual result BEFORE applying this Pull Request
Works.


### Expected result AFTER applying this Pull Request
Works.


### Link to documentations
Please select:
